wlauto/common/android/device.py

```diff
@@ -649,6 +649,10 @@ def get_device_model(self):
         except KeyError:
             return None
 
+    def is_wifi_connected(self):
+        self.logger.debug('Checking for wi-fi connectivity.')
+        return self.execute('dumpsys wifi| grep enabled')
```

Comment:
Could you try and improve this to include a check for connected state?  If we can achieve that then this can go in.